lib/bezier.js

```diff
@@ -727,12 +727,12 @@
       for(i=0; i<len; i++) {
         left = reduced.slice(i,i+1);
         right = reduced.slice(i+2);
-        result = this.curveintersects(left, right);
+        result = this.curveintersects(left, right, null);
```

Comment:
if this is not an actual value, just don't pass it in and let JS do the rest. It'll and up being `undefined` inside the called function.